src/main/drivers/rangefinder/rangefinder_lidarmt.c

```diff
@@ -0,0 +1,96 @@
+/*
+ * This file is part of Betaflight and INAV
+ *
+ * Betaflight and INAV are free software. You can
+ * redistribute this software and/or modify this software under
+ * the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * Betaflight and INAV are distributed in the hope that
+ * they will be useful, but WITHOUT ANY WARRANTY; without even the
+ * implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+ * PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+ * This is a bridge driver between a sensor reader that operates at high level (i.e. on top of UART)
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <string.h>
+
+#include "platform.h"
+
```

Comment:
Maybe add `#ifdef USE_RANGEFINDER_MT` (as we also have USE_RANGEFINDER_MSP) for the remainder of the file (here and #endif at EOF)

Reply:
Done
and tested after the edits

Reply:
Pls update `USE_RANGERFINDER_MT` in instructions in first comment.